server/index.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+let parser = require('body-parser');
+let morgan = require('morgan'); 
+let db = require('./db.js');
+
+let app = express();
+
+app.use(morgan('dev'));
```

Comment:
are you using morgan here?

Reply:
yup! I'm using it to log all the post/gets pings to my server.